main/test_model.py

```diff
@@ -0,0 +1,41 @@
+# Lint as: python3
+"""Apply pre-trained model on test data"""
+
+from src import receive_data
+from src import suite
+import logging
+import pickle
+
+logging.basicConfig(
+    filename="main/test_model.log", filemode="w", level=logging.INFO)
+logging.basicConfig(level=logging.INFO)
+G_data = True
+
+s = suite.Suite()
```

Comment:
since thie file is a test, how about some assertions at the end of it?

Reply:
oh.. I was thinking test_model meaning applying the model on unlabeled data - probably not a good file name. What would be some nice things to assert? Asserting if any data labeled as 1?

Reply:
May I change the file name to `apply_GH_model.py` so that it doesn't sound like a test anymore?

Reply:
You can change the name. Still, since you've got a pre-trained model here, you could feed in some example testing data into a real unit test for it.

Reply:
It's hard to test the process from feeding raw text input -> compute features -> classify, because it requires downloading nltk datasets but I keep getting `readonly` errors. So I fed in some data features and tested the classification results.